module/utils.mjs

```diff
@@ -33,10 +33,12 @@ export function formatModifier(mod) {
  * A helper for using Intl.NumberFormat within handlebars.
  * @param {number} value    The value to format.
  * @param {object} options  Options forwarded to {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat}
+ * @param {boolean} [options.spellOut]  Spell out number as full word, if possible.
```

Comment:
Maybe `wordForm` or `words`? I think 'spell out' is correct it just doesn't feel very... technical to me. I'm not sure.

Reply:
`words` matches well with `numerals`, so I'll go with that.